### ISSUE TITLE
Negative Loss

### ISSUE THREAD

**Issue by say2sarwar** [facebookresearch/simsiam#21]
I am getting Negative loss value during training, is it normal? why did you use negative sign in the loss function?

**Comment by endernewton** [comment]
It is normal, because it negative cosine similarity. The negative sign makes it a proper "loss". Please see details in the paper.
